templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -52,7 +45,7 @@ contents:
     }
 
     persist_nm_conn_files() {
```

Comment:
i believe this function can be removed too. please let me know what other changes would be required

Reply:
Yes, this can be removed.

Reply:
done. thanks!